README.md

```diff
@@ -82,27 +77,47 @@ public:
 
         static void _register_methods() {
            register_method("method", &SimpleClass::method);
+	   
+	   /**
+	    * How to register exports like gdscript
+	    * export var _name = "SimpleClass"
+	    **/
+	   register_property((char *)"base/name", &SimpleClass::_name, Variant("SimpleClass"));
```

Comment:
Updated, was really in hurry, you should wait until godot_header README is updated as creating `.gdns` is left out.